.gitignore

```diff
@@ -108,3 +108,7 @@ __pycache__/
 .pytest_cache/
 .envrc
 report/results.xml
+
+# make lint-fix
+.clj-kondo/
+.lsp/
```

Comment:
something new, where is it coming from? 🤔

Reply:
@qfrank, I couldn't generate these files, which makes sense to me because `.clj-kondo` is the directory generated by the Clojure linter. I don't know how it got generated in status-go. In which case we can remove these changes from `.gitignore`. The directory `.lsp/` I also couldn't generate with `make lint-fix` in status-go.

Reply:
They have wings and flew from status-mobile to status-go 🪰